SyntaxNodeAnalyzer/SyntaxNodeAnalyzer/SyntaxNodeAnalyzer/DiagnosticAnalyzer.cs

```diff
@@ -15,76 +15,80 @@ namespace SyntaxNodeAnalyzer
-        //isEnabledByDefault: Determines whether the analyzer is enabled by default or if the user must manually enable it. Generally set to true
+        // id: Identifies each rule. Same as the public constant declared above
+        // defaultSeverity: Is set to DiagnosticSeverity.[severity] where severity can be Error, Warning, Hidden or Info, but can only be Error or Warning for the purposes of this tutorial
+        // isEnabledByDefault: Determines whether the analyzer is enabled by default or if the user must manually enable it. Generally set to true
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         {
             get
             {
-                //This array contains all the diagnostics that can be shown to the user
+                // This array contains all the diagnostics that can be shown to the user
                 return ImmutableArray.Create(Rule);
             }
         }
 
         public override void Initialize(AnalysisContext context)
         {
-            //Calls the method (first argument) to perform analysis whenever this is a change to a SyntaxNode of kind IfStatement
+            // Calls the method (first argument) to perform analysis whenever this is a change to a SyntaxNode of kind IfStatement
             context.RegisterSyntaxNodeAction(AnalyzeIfStatement, SyntaxKind.IfStatement);
         }
 
-        //This method, which is the method that is registered within Initialize, performs the analysis of the Syntax Tree when an IfStatementSyntax Node is found. If the analysis finds an error, a diagnostic is reported
-        //In this tutorial, this method will walk through the Syntax Tree seen in IfSyntaxTree.jpg and determine if the if-statement being analyzed has the correct spacing
+        // This method, which is the method that is registered within Initialize, performs the analysis of the Syntax Tree when an IfStatementSyntax Node is found. If the analysis finds an error, a diagnostic is reported
+        // In this tutorial, this method will walk through the Syntax Tree seen in IfSyntaxTree.jpg and determine if the if-statement being analyzed has the correct spacing
         private void AnalyzeIfStatement(SyntaxNodeAnalysisContext context)
         {
-            //The SyntaxNode found by the Initialize method should be cast to the expected type. Here, this type is IfStatementSyntax
+            // The SyntaxNode found by the Initialize method should be cast to the expected type. Here, this type is IfStatementSyntax
             var ifStatement = (IfStatementSyntax)context.Node;
 
-            //This statement navigates down the syntax tree one level to extract the 'if' keyword
+            // This statement navigates down the syntax tree one level to extract the 'if' keyword
             var ifKeyword = ifStatement.IfKeyword;
 
+            // Checks if there is any trailing trivia associated with the if-keyword.
```

Comment:
@zoepetard quick definition of trailing trivia, "trailing trivia (e.g. spaces and comments)